src/decorator/live-decorator.ts

```diff
@@ -0,0 +1,19 @@
+import { KalturaLivePlugin } from "../plugin";
```

Comment:
@esakal  this is a naive implementation that does not really do anything. The implementation of logic will be done by other tickets